radixdlt/src/test/java/com/radixdlt/consensus/ValidatingEventCoordinatorTest.java

```diff
@@ -364,4 +365,19 @@ public void when_processing_valid_stored_proposal_and_there_exists_a_new_commit_
 		eventCoordinator.processProposal(proposal);
 		verify(mempool, times(1)).removeCommittedAtom(eq(aid));
 	}
+
+	@Test
+	public void when_processing_get_vertex_request__then_ec_callback_with_response() {
+		Hash vertexId = mock(Hash.class);
+		Vertex vertex = mock(Vertex.class);
+		Consumer<Vertex> callback = mock(Consumer.class);

```

Comment:
This is a compiler warning:
```
Type safety: The expression of type Consumer needs unchecked conversion to conform to Consumer<Vertex>	ValidatingEventCoordinatorTest.java
```
Not really sure it makes sense to mock the trivial consumer.  Could be more easily expressed, and without compiler warnings with:
```
Consumer<Vertex> callback = v -> {};
```

Reply:
I need to verify that the callback is being used below though

Reply:
Could spy?